modules/package.nix

```diff
@@ -181,5 +181,9 @@ in {
       type = listOf (either package path);
       default = [];
     };
+    postUnpack = mkOption {
+      type = nullOr str;
```

Comment:
This should probably be `lines` instead of `str`. That way multiple modules can define it and they'll just get concat'd together.